### PR TITLE
Guidelines on (open) Data

### PR DESCRIPTION
**Pull request by AgnesBelt**
Data guideline, structured as follow:
- Generic intro on open data and data cycle
- Open Science practices pertaining to the release of open data
     - Metadata
     - Referencing
     - Attribution
     - Release
     - License
     - Version control
     - Workflow

@willu47 please let me know how does this look, particularly the references to Version control and Workflow.

### CODE REVIEW

**Review by willu47 — CHANGES_REQUESTED** [review]
Hi @AgnesBelt - looks great. I think it's a good introduction to data outputs, with a clear and concise summary of the basics.

A few thoughts presented inline below.

You could also provide a link to development of a data management plan (for larger projects). For example https://dmponline.dcc.ac.uk/

Also many universities have their own set of public resources about these tools, so it's worth highlighting that researchers can reference these as well.

File: _posts/2022-03-06-data.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- [Referencing](##Referencing) to source material, to provide information regarding where the data come from.
+- Other relevant information, which might describe the methodology used for collecting the data, or the data format provided, or additional documentationi that might be considered relevant to understand the data and their potential use.
+
+## Referencing
+It is important to ensure that original data sources are properly documented and cited, in reference to the data used and processed in the develpent of a derivated data package or dataset used for research. 
+This is needed to ensure that data are traceable.
+Original data sources should be recorded and complete citations for each of the sources used in the generation of a data package or dataset should be listed in the related metadata.
+
+## Attribution
+To ensure all the contributors - i.e. people involved in collecting, processing, structuring and releaseing the data - are acknolwegded, the [CRediT author statement](<https://www.elsevier.com/authors/policies-and-guidelines/credit-author-statement>) can be used as reference.
```
[the remark this answers is not in the document]
Comment (AgnesBelt) [reply _posts/2022-03-06-data.md:58]: Sure, once we have the blog post to link to we can add the reference. Not sure it exists already...?

File: _posts/2022-03-06-data.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+3. **Organizing and Documenting**: documents and data need to be organized in a way that can be understood and taken up by future colleagues or researchers interested in the existing data and work done. 
+More information on how to best ensure so  are available under [Workflow](##Workflow), [Version](##Versioncontrol) [control](##Versioncontrol), and [Metadata](##Metadata).
+
+4. **Storing and Sharing**: as briefly mentioned already in the previous step, data need to be stored on a safe platform, and possibly in a way that can be shared across collaborators and retrieved by the interested audiences. In order to do so, several practices exist, listed here by increasing level of complexity:
+
+    a. *Manually creating a data package*, containing all the necessary information to ensure original data sources, data processing steps and contributors are clearly mentioned. This can be done via tools that provide easy-to-use templates to be filled in with pertainng information, such as the [Data Package Creator](<https://create.frictionlessdata.io/>).
```
[the remark this answers is not in the document]
Comment (AgnesBelt) [reply _posts/2022-03-06-data.md]: I tried and change it according to the structure you suggested. Not sure if the releasing steps of the generic intro should be removed at this point... and just added under storing and sharing.

File: _posts/2022-03-06-data.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Data represent the basis for any research output.
+Data can assume a wide range of formats: from numerical data, to text, to audio, to images.  
+Similarly, data file can consist of a wide range of file formats: from comma-separated values (.csv) files, to text (.txt) files, to image (.jpeg or .png) files.
+
+In order to support the wide reusability of data across different research projects, contexts, and applications, it is important to ensure that data file formats are as generic as possible, i.e. non software specific.
+
+To ensure data are corectly used and managed across research projects, the following process cycle<sup>[1](#myfootnote1)</sup> should be kept in mind:
+![DataCycle](img/DataCycle.png)
+
+1. **Planning**: a plan should be establish in the form of a live document, to identify types of data and related formats that are expected to be used in the project.
```
[the remark this answers is not in the document]
Comment (AgnesBelt) [reply _posts/2022-03-06-data.md]: changed into subheadingss

File: _posts/2022-03-06-data.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+[CC BY licences](<https://creativecommons.org/about/cclicenses/>) are typically used for data licensing. Among CC BY licenses, there are more permissive and more restrictive licenses. Available on the Creative Commons website, [this tool](<https://creativecommons.org/choose/>) can help in choosing the right license for your data.
+
+All licenses that apply to the original data sources should also be considered in selecting the license for the derivated data package or dataset released or shared as reseach output. This is to make sure the derivated work complies with any potential legal restriction imposed on the original data sources.
+
+## Version control
+Version control systems, such as [Git](<https://git-scm.com/>), and dedicated online services, such as [GitHub](<https://github.com/>), allow to record instances of text-based material and (e.g. data saved as .csv or .txt files) and then compare them to visualize differences and changes. They also enable collaborative workflows, thus facilitating researcher to work on the same data simultaneously, save and compare the separate progresses later on.
```
[the remark this answers is not in the document]
Comment (AgnesBelt) [reply _posts/2022-03-06-data.md:104]: I added such empty (for now) references under each main subsection about the open practices.

File: _posts/2022-03-06-data.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+tags: metadata, referencing, attribution, release, licensing, version control, workflow
+---
+
+Data represent the basis for any research output.
+Data can assume a wide range of formats: from numerical data, to text, to audio, to images.  
+Similarly, data file can consist of a wide range of file formats: from comma-separated values (.csv) files, to text (.txt) files, to image (.jpeg or .png) files.
+
+In order to support the wide reusability of data across different research projects, contexts, and applications, it is important to ensure that data file formats are as generic as possible, i.e. non software specific.
+
+To ensure data are corectly used and managed across research projects, the following process cycle<sup>[1](#myfootnote1)</sup> should be kept in mind:
```
[the remark this answers is not in the document]
Comment (AgnesBelt) [reply _posts/2022-03-06-data.md]: Is it ok now?

**Comment by AgnesBelt** [comment]
I did a mess, and added the Attribution guideline under the same branch that I used for the Pull Request related to the Data Guideline. So, now this Pull Request #11 contains both my newly created guidelines for Data and Attribution. Anyway, I guess it's ok for now.

**Comment by willu47** [comment]
Haha, easy to do. If you create a new branch (from the previous branch) containing those commits and then a separate pull request you can partially fix this.

Best practice is to always create a new branch from the main branch when starting work on a new topic which avoids such tangles.

**Comment by AgnesBelt** [comment]
I tried to fix it. I hope I didn't mess it up further now. Attribution guidelines now available in a separate pull request #14

File: _posts/2022-03-06-data.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (willu47) [reply _posts/2022-03-06-data.md:106]: You don't need the `<` and `>` around the hyperlinks in markdown.

**Review by willu47 — COMMENTED** [review]
Suggest adding links to these very useful resources:

[Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html) section of [The Turing Way](https://the-turing-way.netlify.app/welcome.html)

The entry on [data standards for energy modelling](https://forum.u4ria.org/t/data-standards-for-energy-modeling-for-policy-support/25) at the [u4RIA forum](https://forum.u4ria.org)

**Comment by AgnesBelt** [comment]
Hi Will, I have now addressed all your comments. Please, see if it is ready to publish. Thanks!

**Pull request merged**